pygwalker/services/preview_image.py

```diff
@@ -30,21 +30,6 @@ class ChartData(BaseModel):
     n_cols: int = Field(..., alias="nCols")
     title: str
 
-
```

Comment:
For module-level functions and class definitions, it is recommended to separate them with two blank lines.

The action for checking python code style will be added in future.

Reply:
fixed it